### PR TITLE
Login switched to Dialog

### PR DESCRIPTION
**Pull request by monotkate**
Changed the way login works so it uses a dialog over the main page instead of navigating to a separate page.

Combined the loginView controller code into the toolbar controller for the dialog.  Removed loginView.js because it was no longer used.

### CODE REVIEW

**Comment by monotkate** [comment]
Closes #105

**Comment by LongArmMcGee** [comment]
Hawt

**Comment by monotkate** [comment]
@YashdalfTheGray @Mctalian

File: src/index.html [hunk 1]
```diff
@@ -57,10 +56,21 @@
             <h2><span><img ui-sref="home" class="center-img" src="assets/Innovation_Challenge_Bulb.png" width="25"/></span></h2>
             <span class="padding-left-20 idea-likes-click" ui-sref="ideabrowse">All Ideas</span>
             <span flex></span>
-            <md-button class="md-icon-button" aria-label="Account" ng-click="accountClick()">
-                <md-icon md-svg-icon="action:perm_identity" ng-if="!isUserLoggedIn()"></md-icon>
-                <identicon username='username' size='24' ng-if="isUserLoggedIn()" ng-init="getUsername()" class="identicon"></identicon>                
-            </md-button>
+
+            <div class="md-padding dialogdemoBasicUsage" id="popupContainer" ng-cloak="">
```
Comment (YashdalfTheGray) [inline src/index.html]: In this code block, you're using 2 spaces as indentation, change it up to 4 to be consistent.

File: src/navigation/toolbar.js [hunk 2]
```diff
@@ -38,7 +66,68 @@ angular.module('flintAndSteel')
...
+                    controller: DialogController,
+                    templateUrl: 'users/loginView/loginView.tpl.html',
+                    parent: angular.element(document.body),
+                    targetEvent: ev,
+                    clickOutsideToClose: true
+                })
+                .then(function(answer) {
+                    $scope.loginUser(answer);
+                }, function() {
+                    $scope.status = 'You cancelled the dialog.';
```
Comment (YashdalfTheGray) [inline src/navigation/toolbar.js]: Canceled

File: src/index.html (hunk 1 above)
Comment (YashdalfTheGray) [inline src/index.html]: Also, where is the `dialogdemoBasicUsage` class defined?

File: src/users/loginView/loginView.tpl.html [hunk 3]
```diff
@@ -1,29 +1,41 @@
...
+            </md-input-container>
+            <br></br>
+            <md-dialog-actions flex layout="column">
+                <md-button class="md-raised md-primary login-button" ng-click="login(account)" flex>
+                    Login
+                </md-button>
+            </md-dialog-actions>
+            <p></p>
+        </md-dialog-content>
+        <!--<md-dialog-actions>
```
Comment (YashdalfTheGray) [inline src/users/loginView/loginView.tpl.html]: You can just remove this commented out section.

**Comment by monotkate** [comment]
@YashdalfTheGray Should have all your comments fixed now

**Comment by YashdalfTheGray** [comment]
Approved. Waiting on one more.

**Comment by Mctalian** [comment]
I checked it out locally and it looks really good. The only thing I'm a little iffy on is that we blew away an entire test spec. But taking a closer look it only had 3 cases, which rely more on the server doing its job and the last test case was no longer used because we no longer have a sign-up process.

Looks like you have some merge conflicts after I merged in my large changes.

Other than that, I approve.

**Pull request merged**